sensor-vernier/src/main/java/org/concord/sensor/vernier/VernierSensor.java

```diff
@@ -403,20 +403,20 @@ public int setupSensor(int sensorId, SensorRequest request)
 			case SensorID.GDX_MOTION:
 				setUnit("m");
 				setType(QUANTITY_DISTANCE);
-				setName("Pressure");
-				setStepSize(0.034f); //30 samples per second max
+				setName("Motion");
+				setStepSize(0.1f); //30 samples per second max
```

Comment:
Here and elsewhere in this file the comment is out of sync with the actual value.